dict/prh-idiomatic-usage.yml

```
@@ -1522,7 +1522,7 @@ rules:
         to: ボタンを押した
   - expected: "$1　$2"
     pattern:
-      - /(?<!^[#Q][^？！]+)(?!.*[］」])([？！])([^　])/
+      - /(?<!^[#Q][^？！]+)(?!.*[］」])([？！])(?!['"])([^　])/
```

ほぼ同じなのでこだわりはないですが、既存の除外文字列に追加する正規表現もありだと思いました。
動けばよかろう&好みなので、マージしなくて大丈夫です！ 😄 

```suggestion
      - /(?<!^[#Q][^？！]+)(?!.*[］」])([？！])([^　'"])/
```